site/content/events/2015-washington-dc/_email_organizers.txt

```diff
@@ -0,0 +1 @@
+organizers-washington-dc-2015@devopsdays.org
```

Comment:
We've not yet set up either email group. I can set them up, but I need (in email to info@devopsdays.org) what you'd like the email addresses listed for you and Nathen and anyone else to be.

(I'm not too worried about this pre-merge since it's commented out anyhow.)